fflib/src/classes/fflib_SObjectUnitOfWork.cls

```diff
@@ -628,14 +590,54 @@ public virtual class fflib_SObjectUnitOfWork
         }
     }
 
+	private void throwExceptionOnEventSObjectType(String sObjectType)
```

Comment:
@wimvelzeboer nice enhancement. Is it possible to use another name on these methods? Something like checkException<blah> It confuses to me a bit. Name seems to say you will always throw an exception but this is not the case if the condition doesn't match.